src/app/server/OnboardingCodesUtil.cpp

```diff
@@ -45,14 +47,17 @@ void PrintQRCode(chip::RendezvousInformationFlags rendezvousFlags)
             ChipLogProgress(AppServer, "Copy/paste the below URL in a browser to see the QR Code:");
             ChipLogProgress(AppServer, "%s?data=%s", qrCodeBaseUrl, &qrCodeBuffer[0]);
         }
+
+        ChipLogProgress(AppServer, "Manual pairing code: [%s]", manualPairingCode.c_str());
```

Comment:
nit: The line 42 above seems to have two spaces after ":", is there any reason for it?

Reply:
No, fixed